### DISCUSSION TITLE
Adgaurd and Windows AD

### DISCUSSION THREAD

**Discussion by sunnyghat** [AdguardTeam/AdGuardHome#5356] [Q&A]
Hi, 

I was woundering if anyone has a guide on how to integrate Adguard with a Windows AD environment? I run a Windows AD at home, and would like to integrate Adguard into the network to block ad's on some devices.  Although in the DNS server i can set the forwarder to go to the Adguard Server, however this won't allow me to create custom filters for some clients.

I was thinking maybe adding:
`[/ghataura.net/]10.100.1.1
[/ghataura.net/]fd99:9971::10:100:1:1`

I have therefore setup my Windows DHCP server to set the Adguard server as the default DNS server. and add to the upstream DNS servers, were  10.100.1.1 and fd99:9971::10:100:1:1 are the IP addresses of my Windows AD DNS server, but on some devices (linux in particular) they do not resolve hosts on the AD network (ghataura.net).

Am i missing something in the configuration ?

**Comment by donald2612** [comment]
Nothing missing, works here that way fantastic in my home-AD. I would start using IPv4-only and troubleshoot the query-log, when 'diggin' from the linux boxes.
Some private reverse entries shouldn't be missing in the rDns-section neither like 
[/168.192.in-addr.arpa/]192.168.0.63:53
